frontend/src/containers/GetSurveysList.js

```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux'
+import SurveysList from '../components/SurveysList';
+
+
+const getSurversFromServerSide = (server) =>{
+    server.fetchDev(10);
```

Comment:
Don't put magic numbers in your code. Use local named constants, global named constants or even config parameters.